src/depot/outdated/update.clj

```diff
@@ -62,13 +78,15 @@
 
   `loc` points at the map."
   [loc consider-types repos]
-  (rzip/up
-    (loop [loc (rzip/down loc)]
-      (let [loc' (try-update-artifact loc consider-types repos)
-            loc'' (rzip/right (rzip/right loc'))]
-        (if loc''
-          (recur loc'')
-          loc')))))
+  (if-let [first-key (rzip/down loc)]
```

Comment:
It's not like I had this pattern [on a slide](http://arnebrasseur.net/talks/2018-clojure-zip-denver/#54) just a few weeks ago, warning people of the caveats of zippers.

Reply:
Haha, well I'm glad we have the resident zipper expert on the case :smile: